### PR TITLE
Adding dep tool to readme, fixing makefile name

### PR DESCRIPTION
**Pull request by secondsun**
Minor changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,19 @@
 ## Prerequisites
 
 * Golang (1.10)
+ * [Dep tool](https://golang.github.io/dep/docs/installation.html)
+* [oc tools >= 3.9.0](https://github.com/openshift/origin/releases)
 * Nodejs
```
Comment (aidenkeating) [inline README.md]: Do we know if there's a minimum version we require? Maybe we could link out to the install guides for Go and Node too?

Comment (secondsun) [reply README.md]: The answer to "What is the minimum number we require" is "What is the minimum we want to support".

Comment (aidenkeating) [reply README.md]: Good question. Not one I know the answer to. But as Node 8 is the current active LTS release that could be a safe bet for now? https://github.com/nodejs/Release#release-schedule

**Comment by wei-lee** [comment]
@secondsun Unfortunately, due to security reasons, this project on CircleCI won't build PRs from forks. We will have to create PRs from the upstream repo for now.

**Comment by wei-lee** [comment]
@secondsun can you recreate the PR from upstream if you still want to merge this one?

**Comment by secondsun** [comment]
@wei-lee Will do

**Pull request closed without merging**